models/comments.model.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../db/connection')
+
+exports.deleteCommentById = (comment_id) => {
+    return db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *;`, [comment_id])
+        .then(() => {
+            return
```

Comment:
we could catch the error here without checking the comment. As when a comment is deleted we get the deleted comment returned back. If the rows.length were 0 we could Promise.reject here :) Your way works which is great, but a simplier option if you wanted :)